### PR TITLE
Add delete instance method and spec

### PR DESCRIPTION
**Pull request by mlan080**

### CODE REVIEW

**Comment by mlan080** [comment]
Question: Why is my delete branch adding all my previous commits? I just want to add my
*def delete 
*describe '#delete'

File: app/models/shop.rb [hunk 1]
```diff
@@ -24,4 +30,18 @@ def self.last
...
+    h = DB[:shops].first(:id => id) #sequel doc uses first for retrieving record
+    return nil if h == nil
+    Shop.new(h) #returns a shop object with hash argument
+  end
+
+  def set(id, name)
+    DB[:shops].where(id: id).update(name: name)
+  end
+
+  def delete(id)
```
Comment (lucasrenan) [inline app/models/shop.rb]: if you add a method reader for `id` then you don't need to pass `id` as a parameter :)

**Comment by lucasrenan** [comment]
btw, this PR is duplicated: https://github.com/mlan080/tea-shops/pull/25

File: app/models/shop.rb (hunk 1 above)
Comment (mlan080) [reply app/models/shop.rb]: So do I need to pass ID like this? 

attr_accessor :id, :name, :description

Comment (lucasrenan) [reply app/models/shop.rb]: you can follow the same concept used for the `update` method

**Pull request closed without merging**